packages/code/src/CopyButton/CopyButton.tsx

```diff
@@ -39,31 +51,70 @@ function CopyButton({ onCopy, contents }: CopyProps) {
     }
 
     return () => clipboard.destroy();
-  }, [buttonNode, contents, copied, portalContainer]);
+  }, [buttonRef, contents, copied, portalContainer]);
```

Comment:
`buttonRef` won't trigger a rerender since it's a ref, that's probably why a callback ref was initially used. Although, it doesn't seem like it needs to be a dependency. Was the callback ref causing issues?

Reply:
no, that wasn't causing issues, but I wasn't able to click the button with the enter/space keys so I defined a separate ref